bioio_ome_zarr/writers/config.py

```diff
@@ -0,0 +1,106 @@
+from typing import Any, Dict, Union
+
+import dask.array as da
+import numpy as np
+
+from .utils import chunk_size_from_memory_target
+
+
+def get_ome_zarr_writer_config_for_viz(
+    data: Union[np.ndarray, da.Array],
+) -> Dict[str, Any]:
+    """
+    Generate a default OmeZarrWriterV3 config for visualization use.
```

Comment:
Explaining some of the choices we made would be good for the user - ex. how we want the chunk sizes to be optimized for what will fit easily into memory in Vol-E

Reply:
I added descriptions of the choices but am not really sure how to go about explaining them.